pcsx2/SIO/Pad/PadDualshock2.cpp

```diff
@@ -95,6 +95,79 @@ static const SettingInfo s_settings[] = {
 const Pad::ControllerInfo PadDualshock2::ControllerInfo = {Pad::ControllerType::DualShock2, "DualShock2",
 	TRANSLATE_NOOP("Pad", "DualShock 2"), s_bindings, s_settings, Pad::VibrationCapabilities::LargeSmallMotors};
 
+void PadDualshock2::ConfigLog()
+{
+	const auto [port, slot] = sioConvertPadToPortAndSlot(unifiedSlot);
+	std::string smallMotorStr;
```

Comment:
This and largeMotorStr could be `std::string_view` (since the values are literals), to avoid a potential heap allocation if it doesn't fit in the small string buffer.